documentation/book/assembly-upgrade.adoc

```diff
@@ -13,10 +13,13 @@ When a newer version of {ProductName} is available, it may provide support for n
 Therefore, you will need to upgrade to the new version of {ProductName} before you can upgrade to a higher supported Kafka version.
 Upgrading the version of {ProductName} is done by upgrading the Cluster Operator deployment to the new version.
 
+//Upstream only
```

Comment:
Could you please not use the "upstream" and "downstream" comments?

Reply:
Sure, I've removed them.